src/isReply.js

```diff
@@ -0,0 +1,15 @@
+const isReply = (tweet) => {
+    var RT = /^RT/i;
+    
+    if (RT.test(tweet.text) ||
```

Comment:
I'm guessing you have tested `.test(` on your own bot successfully?

Reply:
Yes... It filters out tweets that begin with "RT".

Reply:
Should I take that out?